packages/sql-graphql/lib/entity-to-type.js

```diff
@@ -50,10 +50,11 @@ function constructGraph (app, entity, opts, ignore) {
     /* istanbul ignore next */
     if (field.enum) {
       const enumValues = field.enum.reduce((acc, enumValue) => {
-        acc[enumValue] = { value: enumValue }
+        acc[enumValue.replace(/-/g, '_')] = { value: enumValue }
```

Comment:
Hey! 👋🏼 
Regarding [this issue](https://github.com/platformatic/platformatic/issues/507), can we replace a list of characters, including `/`, with `_`? I think it will solve a lot of potential issues we can have in the future...

Reply:
```suggestion
        acc[enumValue.replace(/[^\w\s]/g, '_')] = { value: enumValue }
```